app/views/poems/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= simple_form_for (@poem) do |form| %>
+  <%= render 'errors', object: @poem %>
+      
+  <%= form.input :title %>
+  <%= form.select :status,
+        ["Available", "Pending", "Accepted – Forthcoming", "Published"]
+  %>
+  <%= form.input :user_id, 
+        as: :hidden,
+        input_html: { value: current_user.id }
```

Comment:
Typically, we indent one extra level in regards to the previous line, which includes the `<%=`.